content/operate/rc/changelog/december-2024.md

```diff
@@ -18,4 +18,12 @@ weight: 38
 
 You can now add a user with the **Logs Viewer** role in the [Access Management]({{< relref "/operate/rc/security/access-control/access-management" >}}) screen. Logs Viewers can only use the [Redis Cloud API]({{< relref "/operate/rc/api" >}}) [`GET logs/`](https://api.redislabs.com/v1/swagger-ui/index.html#/Account/getAccountSystemLogs) endpoint. 
 
-See [Team Management roles]({{< relref "/operate/rc/security/access-control/access-management#team-management-roles" >}}) to see an overview of user roles and their permissions.
\ No newline at end of file
+See [Team Management roles]({{< relref "/operate/rc/security/access-control/access-management#team-management-roles" >}}) to see an overview of user roles and their permissions.
+
+### Redis Flex preview on Redis Cloud Essentials
+
+Redis Flex is now available in Preview on Redis Cloud Essentials.
+
+Redis Flex databases have a tiered solid state drive (SSD) and RAM architecture. Using SSDs instead of RAM significantly reduces infrastructure costs, which means developers can build applications that require large datasets using the same Redis API.
```

Comment:
are we using RAM or DRAM in other references on the docs?

Reply:
I adapted this wording from the current Auto tiering docs, which uses RAM, not DRAM. I can replace it if it makes sense.